travis_ci.yaml.example

```diff
@@ -0,0 +1,2 @@
+---
+token: "2343a00b02d343e0f01ca"
```

Comment:
Maybe better to put some other "fake" value in here so all those bots which automatically scan Github repos for credentials won't annoy us that we have "valid" credentials in the repo.